XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -396,6 +396,11 @@ public Vector2 NudgeConstraints
         /// </summary>
         public bool IsRotating => !updatedAngle.Equals(0f);
 
+        /// <summary>Gets or sets a value indicating whether this instance can be moved in space.</summary>
+        /// <value>
+        ///   <c>true</c> if this instance is translate locked; otherwise, <c>false</c>.</value>
+        public bool IsTranslateLocked { get; set; } = false;
```

Comment:
Should we add a serialized field here to configure it in the inspector?

Reply:
fixed.